typespec/Applications.Core/environments.tsp

```diff
@@ -77,23 +78,29 @@ model EnvironmentProperties {
 model RecipeConfigProperties {
   @doc("Configuration for Terraform Recipes. Controls how Terraform plans and applies templates as part of Recipe deployment.")
   terraform?: TerraformConfigProperties;
+
+  @doc("Environment variables injected during Terraform Recipe execution for the recipes in the environment.")
+  envVariables?: EnvironmentVariables;
```

Comment:
This should be `env` like the design doc. https://github.com/radius-project/design-notes/pull/39/files#diff-f9f52b8c3872a6c388dff224c472dfdd39bb011c139480ed60e7f4facd68aa10R183

We review API changes in detail because they are hard to change later. Please do a pass on on this and make sure that the API changes match the design doc.

Reply:
I think this drifted from design doc after https://github.com/vishwahiremat/radius/pull/1#discussion_r1500189753 @lakshmimsft thoughts?

Reply:
If there is drift from an approved design doc, then it needs to come back to the design meeting. We review details like naming for clarity and consistency.

Reply:
> If there is drift from an approved design doc, then it needs to come back to the design meeting. We review details like naming for clarity and consistency.

Completely agree. This was a miss.